src/wcs1_0/src/test/java/org/geoserver/wcs/GetCoverageTest.java

```diff
@@ -222,81 +222,25 @@ public void testWorkspaceQualified() throws Exception {
     }
 
     @Test
-    public void testGEOS6540_1() throws Exception {
+    public void testGEOS6540() throws Exception {
         String queryString =
                 "wcs?sourcecoverage="
                         + getLayerId(MOSAIC)
                         + "&request=getcoverage"
-                        + "&service=wcs&version=1.0.0&format=ArcGrid&crs=EPSG:4326"
+                        + "&service=wcs&version=1.0.0&format=image/geotiff&crs=EPSG:4326"
```

Comment:
These tests were checking for the `format` parameter being case-insensitive. 

See https://osgeo-org.atlassian.net/browse/GEOS-6540

Reply:
I'm going to rename the test methods to make that clear.